frontend/src/data_context.tsx

```diff
@@ -115,6 +116,12 @@ export function WithDataContext(props: {children: React.ReactNode}) {
   async function getUsers() {
     return fetchAndSet('/api/manage/users', setUsers, 'users')
   }
+  /**
+   * Fetches all legal colors for all genebanks.
+   */
+  async function getColors() {
+    return fetchAndSet('/api/colors', setColors)
+  }
```

Comment:
I tested now and in color i only get "no option" Should this be populated by allowed colors for Gotlands kanin?

Reply:
that's so weird, but the `/api/colors` endpoint disappeared. I re-added it, but I have no idea where it went...